projects/ng-live-docs/src/documentation.ts

```diff
@@ -92,11 +96,50 @@ export const Documentation = {
      * Returns angular routes used when displaying the documentation/examples for the components.
      */
     getRoutes(): Routes {
-        return Documentation.getAllEntries().map((documentationEntry: DocumentationEntry) => ({
+        const routes = Documentation.getAllEntries().map((documentationEntry: DocumentationEntry) => ({
             path: documentationEntry.urlSegment,
             component: DocumentationContainerComponent,
-            data: { documentationEntry },
+            // Add three tabs as subroutes
+            children: [{
+                path: 'documentation',
+                component: DocumentationContainerDocumentationComponent,
+                data: { component: documentationEntry.component }
+            },
+            {
+                path: 'api',
+                component: DocumentationContainerApiComponent,
+                data: { component: documentationEntry.component }
+            },
+            {
+                path: 'example',
+                component: DocumentationContainerExampleComponent,
+                data: { examples: documentationEntry.examples ? documentationEntry.examples : [] },
+                children: documentationEntry.examples?.length ? [
+                    ...documentationEntry.examples.map((exampleEntry: ExampleEntry) => ({
+                        path: exampleEntry.component.name.toLowerCase(),
```

Comment:
We should avoid using `Function.name` because functions can get renamed by optimizers. That means that in production, a function named `hello()` may be minified as `a_()` and its name would become `a_`. Also it doesn't look very good in the url. I think we may need a breaking change, that is, introduce a `urlSegment` on each example entry (which means we need to add that to every example in our application :(

For a component named `DatagridColumnWidthExampleComponent`, its url segment should be `datagrid-column-witdth`